doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -96,6 +96,13 @@ ESB
 
 * Fixed the ``update_radio_crc()`` function in order to correctly configure the CRC's registers (8 bits, 16 bits, or none).
 
+Wi-Fi
```

Comment:
Not sure if we should add this here as no Protocol has been added.

Reply:
nrf7002 introduces new Wi-Fi protocol that can be used by applications.